esileapclient/osc/v1/offer.py

```diff
@@ -107,7 +107,9 @@ def get_parser(self, prog_name):
             '--status',
             dest='status',
             required=False,
-            help="Show all offers with given status.")
+            help="Show all offers with given status. "
+                 "Use --status 'any' to show offers with any status "
+                 "(by default only offers with 'available' status are shown).")
```

Comment:
That's not quite right: https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/api/controllers/v1/offer.py#L140